droidlet/interpreter/robot/default_behaviors.py

```diff
@@ -101,3 +106,28 @@ def explore(agent):
     x,y,t = agent.mover.get_base_pos()
     goal = get_distant_goal(x,y,t)
     start_explore(agent, goal)
+
+def get_task_data(agent):
+    with open(agent.opts.reexplore_json, 'r') as f:
+        reex = json.load(f)
+
+    reexplore_id = get_unique_val_from_memory(agent, 'reexplore_id', int) 
+
+    if str(reexplore_id) not in reex.keys():
```

Comment:
can you `reex_key = str(reexplore_id)` and use that here and below. repeating `str(reexplore_id)` is giving me a mild heart-attack about typos.

Reply:
yes.

Reply:
yes.